src/components/filters/useOperatingSystemFilter.js

```diff
@@ -9,14 +9,20 @@ export const operatingSystemFilterReducer = (_state, { type, payload }) => ({
     }
 });
 
-export const useOperatingSystemFilter = ([state, dispatch] = [operatingSystemFilterState]) => {
+export const useOperatingSystemFilter = (
+    operatingSystems,
+    operatingSystemsLoaded,
+    [state, dispatch] = [operatingSystemFilterState]
+) => {
+    // FIXME: is this state value needed and used at all?
```

Comment:
I also think that this can be removed.

Reply:
why? I still see this state variable unused and wondering whether this is needed at all

Reply:
I believe that this state variable is used as a default value holder. But this is not needed if we are always getting `operatingSystemFilter` from the state prop (I think we are getting this always).

Reply:
This state unused (in Inventory). I believe it was meant as a default for when the hook would be used outside the Inventory without redux. There doesn't appear to be any other app using it though. We should double check though.

Reply:
Can we leave this FIXME comment, so that we don't forget later, after this PR, make a check? :-)